config/lava/chromeos/chromiumosvm-tast-template.jinja2

```diff
@@ -0,0 +1,39 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block main %}
+{{ super () }}
+{% endblock %}
+
+{% block actions %}
+{{ super () }}
+
+actions:
+- deploy:
+    images:
+      bios:
+        image_arg: -bios {bios}
+        url: http://storage.kernelci.org/images/uefi/edk2-stable202005/x86_64/OVMF.fd
+      rootfs:
+        image_arg: -drive format=raw,file={rootfs},id=lavatest2,if=none
+        url: {{ rootfs_url }}chromiumos_test_image.bin.gz
+        compression: gz
+{%- if kernel_url %}
+      kernel:
+        image_arg: -kernel {kernel} -append "init=/sbin/init boot=local rootwait ro noresume noswap loglevel=7 earlyprintk=serial,keep console=tty0 console=ttyS0 i915.modeset=1 cros_efi cros_debug root=/dev/sda3"
+        url: {{ kernel_url }}
+{%- endif %}
```

Comment:
We only just got one QEMU job running in KernelCI, and it's not booting with linux-next (and no modules installed yet). So I think this part should be left out for now to keep using the built-in kernel instead until we have the rest sorted out.

Reply:
Done, removed it.